src/user-util/index.ts

```diff
@@ -45,6 +45,87 @@ ${el}`;
+// Why is this not pointing to `main`?
+//   So that if the docs are moved around or renamed in the future, the links in previous PT versions still work
+// Does this need to be updated before every release?
+//   No, only when the docs are changed
+const docsVersion = 'v2.0.0';
+const customizeDocsMessage = `You can customize this check by setting the targetSize option, more details at https://github.com/cloudfour/pleasantest/blob/${docsVersion}/docs/errors/target-size.md`;
+
+export const assertTargetSize = (
+  el: Element,
+  targetSize: number | true | undefined,
+) => {
+  const { width, height } = el.getBoundingClientRect();
+
+  const display = getComputedStyle(el).display;
+
+  // Per the W3C recommendation, inline elements are excluded
+  // See: https://www.w3.org/WAI/WCAG21/Understanding/target-size.html
+  if (display === 'inline') {
+    return;
+  }
+
+  const size = typeof targetSize === 'number' ? targetSize : 44;
+
+  if (width < size || height < size) {
+    if (el instanceof HTMLInputElement && el.type === 'checkbox') {
+      const labelSize = el.labels?.[0]?.getBoundingClientRect();
+
+      // Checkbox did not have label
+      if (!labelSize) {
+        const targetSizeMsg =
+          typeof targetSize === 'number'
+            ? `Target size of checkbox is smaller than ${size}px × ${size}px`
+            : 'Target size of checkbox does not meet W3C recommendation of 44px × 44px: https://www.w3.org/WAI/WCAG21/Understanding/target-size.html';
+
+        throw error`Cannot click checkbox that is too small.
+${targetSizeMsg}
+Element was ${width}px × ${height}px
+${el}
+You can increase the target size of the checkbox by adding a label that is larger than ${size}px × ${size}px
+${customizeDocsMessage}`;
+      }
+
+      // If label is valid
+      if (labelSize.width >= size && labelSize.height >= size) {
+        return;
+      }
+
+      // Checkbox and label was too small
+      const targetSizeMsg =
+        typeof targetSize === 'number'
+          ? `Target size of element is smaller than ${size}px × ${size}px`
+          : 'Target size of element does not meet W3C recommendation of 44px × 44px: https://www.w3.org/WAI/WCAG21/Understanding/target-size.html';
+
+      throw error`Cannot click checkbox that is too small.
+${targetSizeMsg}
+Checkbox was ${width}px × ${height}px
+${el}
+Label associated with the checkbox was ${labelSize.width}px × ${
+        labelSize.height
+      }px
+${el.labels![0]}
+You can increase the target size by making the label or checkbox larger than ${size}px × ${size}px.
+${customizeDocsMessage}`;
+    }
+
+    // Non-checkbox case
+    const targetSizeMsg =
+      typeof targetSize === 'number'
+        ? `Target size of element is smaller than ${size}px × ${size}px`
+        : 'Target size of element does not meet W3C recommendation of 44px × 44px: https://www.w3.org/WAI/WCAG21/Understanding/target-size.html';
```

Comment:
Would it be worth DRYing up these repeated error messages to make them easier to maintain?

Reply:
I'm looking into this now. 🔍 👀

Reply:
Done https://github.com/cloudfour/pleasantest/pull/248/commits/38de7787ed5ed6456da7fe099608b8724b3e358a